src/app/components/Select/index.tsx

```diff
@@ -52,7 +51,6 @@ const StyledListbox = styled('ul')(({ theme }) => ({
 const StyledOption = styled(OptionUnstyled)(({ theme }) => ({
   boxSizing: 'border-box',
   listStyle: 'none',
-  height: '36px',
   padding: `${theme.spacing(3)} ${theme.spacing(4)}`,
```

Comment:
(if it's meant to be smaller we should adjust padding instead)

Reply:
Don't particularly like the solution, as the Button and the Option have different heights now. https://github.com/oasisprotocol/explorer/pull/72

Reply:
Ah, if they should have the same height I'd keep height, remove vertical padding, and center text. Updated PR